django/cantusdb_project/main_app/templates/edit_volpiano.html

```diff
@@ -0,0 +1,215 @@
+                </small>
+            {% else %}
+                <h3>Full text &amp; Volpiano edit form</h3>
+                <br>
+                <dl>
+                    <dd><small>1) Select a <b>folio</b> or a <b>feast</b> (in the right block)</small></dd>
+                    <dd><small>2) Click <b>"EDIT"</b> next to any chant</small></dd>
+                    <dd><small>3) The <b>fulltext</b> and <b>Volpiano</b> fields should appear in this area</small></dd>
+                    <dd><small>4) Edit the fields <b>according to the manuscript, following the fulltext guidelines created by Cantus</b></small></dd>
+                    <dd><small>5) Click <b>"SAVE"</b></small></dd>
+                </dl> 
+            {% endif %}
+        </div>
+
+        <div class="col p-0">
+            <!-- global search bar-->
+            {% include "global_search_bar.html" %}
+
+            <div class="card mb-3 w-100">
+                <div class="card-header">
+                    <h4>{{ source.siglum }}</h4>
+                </div>
+                <div class="card-body">
+                    <small>
+                        <!--a small selector of all folios of this source-->
+                        <select id="folioSelect" class="w-30">
+                            <option value="">Select a folio:</option>
+                            {% for folio in folios %}
+                                <option value="{{ folio }}">{{ folio }}</option>
+                            {% endfor %}
+                        </select>             
+                        
+                        {% if previous_folio %}
+                            <a href="{% url "source-edit-volpiano" source.id %}?folio={{ previous_folio }}">{{ previous_folio }} <</a>
+                        {% endif %}
+                        {% if next_folio %}
+                            &nbsp;<a href="{% url "source-edit-volpiano" source.id %}?folio={{ next_folio }}">> {{ next_folio }}</a>
+                        {% endif %}             
+
+                        <br>
+
+                        <select id="feastSelect">
+                            <option value="">Select a feast:</option>
+                            {% for folio, feast in feasts_with_folios %}
+                                <option value="{{ feast.id }}">{{ folio }} - {{ feast.name }}</option>
+                            {% endfor %}
+                        </select>
+
+                        <br>
+
+                    </small>
+                    {% if chants.all %}
+                    <table class="table table-sm small table-bordered">
+                        <tbody>
+                            {% for chant in chants %}
+                                <tr>
+                                    <td class="text-wrap" style="text-align:center">
+                                        <small>{{ forloop.counter }}</small>
```

Comment:
This is likely the `sequence_number` field of the chant model, instead of just a loop counter.